Sources/Commands/SwiftTestTool.swift

```diff
@@ -521,3 +530,12 @@ struct TestSuite {
         })
     }
 }
+
+
+extension Sequence where Iterator.Element == TestSuite {
+    var allTests: [String] {
+        return flatMap { $0.tests }.flatMap{ testCase in
+            testCase.tests.map{ testCase.name + "/" + $0 }
+        }
```

Comment:
We've adopted the popular convention of using parenthesis in closures (`({ })`), if there are no side-effects.

```swift
return flatMap({ $0.tests }).flatMap({ testCase in
    testCase.tests.map({ testCase.name + "/" + $0 })
})
```